src/LibraryManager.Vsix/ErrorList/ErrorList.cs

```diff
@@ -37,7 +38,7 @@ public bool HandleErrors(IEnumerable<ILibraryInstallationResult> results)
                     foreach (IError error in result.Errors)
                     {
                         Logger.LogEvent(error.Message, LogLevel.Operation);
-                        Telemetry.TrackOperation("error", new KeyValuePair<string, object>("code", error.Code));
+                        Telemetry.TrackOperation("error", TelemetryResult.Failure, new KeyValuePair<string, object>("code", error.Code));
```

Comment:
Not sure if the operation name "error" is best, as it doesn't say which operation failed.  Can we know which operation it was?

Reply:
Yes, I agree with you but did not want to change what was in place without understanding more about how we should be logging telemetry events in general for this feature. I will log a bug to review Telemetry in general for LibMan.